app/models/grda_warehouse/service_history_enrollment.rb

```diff
@@ -233,11 +233,17 @@ def self.service_types
 
   # Category 3 is "Homeless only under other federal statuses" and
   # is defined as a housing status of value 5
+  # TODO: updated this for FY2024 to not use cached values,
+  # this appears to only be used in the SPM, so maybe we can just tear it out and use
+  # source data there instead
   scope :category_3, -> do
-    where(
-      arel_table[:housing_status_at_entry].eq(5).
-        or(arel_table[:housing_status_at_exit].eq(5)),
-    )
+    # pre FY2024 logic, remove after we confirm the new version works
+    # where(
+    #   arel_table[:housing_status_at_entry].eq(5).
+    #     or(arel_table[:housing_status_at_exit].eq(5)),
+    # )
+    left_outer_joins(enrollment: :exit).
+      where(e_t[:LivingSituation].eq(205).or(ex_t[:HousingAssessment].eq(205)))
```

Comment:
Same here… there is no way 205 for W5.1?

Reply:
Right, that should have been 5 I believe, thanks!